backstage.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "1.14.2"
+  "version": "1.16.0"
 }
```

Comment:
For the upgrade path, I followed the instructions on https://backstage.github.io/upgrade-helper/?from=1.14.2&to=1.16.0